### ISSUE TITLE
INDI not starting on 8GB RPi4b setup

### ISSUE THREAD

**Issue by taldridge** [rlancaste/AstroPi3#24]
Did an install on 128GB USB3 stick using Ubuntu-Mate for ARMH.  When running EKOS from KSTARS menu the INDI is starting never completes and no devices are mounted.  Further, the INDI Web Manager on the desktop will not start as the application is not found.  Same results for Ubuntu-Mate for ARM64.

My devices are physically connected as follows:
CCD: RPi HQ camera direct to CSI port
Guide CCD: RPi HQ camera via Arducam adapter to USB3 port
4 port powered hub connected to USB2.0 port
DSC via CP2102 adapter to hub
GPUSB to hub
ZWO EAF to hub

**Comment by rlancaste** [comment]
More information would be needed to see what could be wrong.  My script just simply runs commands to install the various applications and configure various settings.  If there is an issue with the script, we would need to know what didn't get installed properly or what setting didn't get set correctly.  If there is an issue with KStars or INDI, we should address that on the indi forum or a bug report.  I can help you with those too, but first we need to know what is wrong.  Or possibly the script ran correctly and things were just installed differently?  Based on what you said here, it sounds like INDI and indiweb were possibly not installed correctly, or maybe they were installed in a different location than where they were expected to be?  Is your system running off the usb stick or did you just use an installer for Ubuntu off the usb stick?

**Comment by taldridge** [comment]
Thanks for getting back to me.  The system is running from the USB stick.  I’ll see if I can get more data for this issue.Tomm
    	Get Outlook for iOS
    
  

 From: rlancasteAstro ***@***.***>Sent: Saturday, September 3, 2022 9:00 AMTo: rlancaste/AstroPi3 ***@***.***>Cc: Tomm Aldridge ***@***.***>; Author ***@***.***>Subject: Re: [rlancaste/AstroPi3] INDI not starting on 8GB RPi4b setup (Issue #24) 
More information would be needed to see what could be wrong  My script just simply runs commands to install the various applications and configure various settings.  If there is an issue with the script, we would need to know what didn't get installed properly or what setting didn't get set correctly.  If there is an issue with KStars or INDI, we should address that on the indi forum or a bug report.  I can help you with those too, but first we need to know what is wrong.  Or possibly the script ran correctly and things were just installed differently?  Based on what you said here, it sounds like INDI and indiweb were possibly not installed correctly, or maybe they were installed in a different location than where they were expected to be?  Is your system running off the usb stick or did you just use an installer for Ubuntu off the usb stick?

**Comment by rlancaste** [comment]
Yes that is probably part (or all) of the problem right there, you might need to change the INDI settings in kstars for the path to INDI since it would not be the default.  The same kind of issue might be true for the indiweb link on the desktop.  First, though, make sure they are actually installed.

**Comment by taldridge** [comment]
Thanks, I will try to resolve the correct path and fix.
    	Get Outlook for iOS
    
  

 From: rlancasteAstro ***@***.***>Sent: Saturday, September 3, 2022 10:28 AMTo: rlancaste/AstroPi3 ***@***.***>Cc: Tomm Aldridge ***@***.***>; Author ***@***.***>Subject: Re: [rlancaste/AstroPi3] INDI not starting on 8GB RPi4b setup (Issue #24) 
Yes that is probably part (or all) of the problem right there, you might need to change the INDI settings in kstars for the path to INDI since it would not be the default.  The same kind of issue might be true for the indiweb link on the desktop.  First, though, make sure they are actually installed.

**Comment by taldridge** [comment]
I checked the paths in K stars INDI setup and they point to the right places.  What next?
